packages/components/pie-cookie-banner/test/component/pie-cookie-banner.spec.ts

```diff
@@ -106,38 +100,38 @@ test.describe('PieCookieBanner - Component tests', () => {
                 },
             });
 
-            const cookieBanner = page.locator(componentSelector);
-
             // Act
-            await page.click(elementSelector);
-            const modal = page.locator(modalSelector);
+            await pieCookieBannerComponent.clickManagePreferencesAll(elementLevel.name);
+
+            const modal = pieModalComponent.isModalVisible();
+            const cookieBanner = await pieCookieBannerComponent.isCookieBannerVisible();
 
             // Assert
-            expect(modal).toBeVisible();
-            expect(cookieBanner).not.toBeVisible();
+            expect.soft(modal).toBeTruthy();
+            expect.soft(cookieBanner).toBeFalsy();
```

Comment:
I think we've introduced some ambiguity here. The previous test was very easy to understand in it's assertions and this seems to be a lot less clear.

Why do we check (softly) that the cookiebanner is falsy?

Reply:
Ideally soft checks should be made before the final assert as sometimes you can get into a situation where if you have multiple `expect` assertions in a test, you can easily lose track as to which of these has caused an error and can also fail early. Ideally, you want to be doing soft asserts before the final hard assert as then if one of these tests were to fail, you'll easily be able to identify which of these assertions are causing the failure.

@siggerzz and I have also discussed about the ambiguity of our testing names with use of `falsy` and `truthy` and have both agreed for it to be something we should look into in the future as we'd much rather be able to have page objects but also use the native playwright functions in the tests.